### PR TITLE
Feature/189 install maykin 2fa

### LINKED ISSUE #189: Replace two_factor with maykin_2fa
I thought that this already happened during the Django 4.2 upgrade, but it seems we're still using our django-two-factor-auth fork (this probably also applies to the other components)

https://maykin-2fa.readthedocs.io/en/stable/

### PR DESCRIPTION
**Pull request by Coperh**
Fixes #189

### CODE REVIEW

**Comment by Coperh** [comment]
Requires maykinmedia/open-api-framework#27 to be merged and released

**Comment by Coperh** [comment]
Requires https://github.com/maykinmedia/open-api-framework/pull/31 to be merged and released.

**Comment by Coperh** [comment]
Waiting for Other PRs to merge

File: src/openklant/accounts/tests/test_admin.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+LOGIN_URL = reverse_lazy("admin:login")
+
+
+@override_settings(
+    USE_OIDC_FOR_ADMIN_LOGIN=False,
+    MAYKIN_2FA_ALLOW_MFA_BYPASS_BACKENDS=[],  # enforce MFA
+)
+class RecoveryTokenTests(WebTest):
+    @tag("gh-4072")
```
Comment (Coperh) [inline src/openklant/accounts/tests/test_admin.py:17]: Do we care enough to remove this tag?

Comment (stevenbal) [reply src/openklant/accounts/tests/test_admin.py:17]: Hmm yeah, let's just remove it

**Review by stevenbal — CHANGES_REQUESTED** [review]
Works as expected, could you also add `DISABLE_2FA` to the docker-compose envvars (with default `True`)?

I do notice that some templates show the admin version though, these should not be visible until the user is fully authenticated

after adding 2FA:
image
backuptokens template:
image

I think this can be fixed by overriding the base template footer: https://github.com/open-zaak/open-zaak/blob/main/src/openzaak/templates/maykin_2fa/base.html

**Review by stevenbal — APPROVED** [review]

**Pull request merged**